README.md

```diff
@@ -10,7 +10,19 @@ for OpenShift Console.
 
 ## Local development
 
-### Option 1 (recommended): Docker + VSCode Remote Container
+### Option 1 (recommended):
+
+In `kubevirt-plugin` directory run the following (Make sure you are logged in):
```

Comment:
You may want to point out that you'll need two separate terminals to run these. One for `yarn dev` and another for `yarn start-console`. 

It would probably be cleaner to address each terminal separately. Something like:

Open two terminals and navigate to the kubevirt-plugin directory in both of them. The first terminal will run a containerized instance of console and the second will run the kubevirt-plugin.

In the first terminal:

1. Log into the OpenShift cluster you are using.
2. Run `yarn start-console` OR `./start-console.sh` OR `./start-console-auth-mode.sh` 

In the second terminal:

1. Run `yarn && yarn dev`

Reply:
Great suggestion, thanks :)